### PR TITLE
0.17 methods

### PR DESCRIPTION
**Pull request by juscamarena**
I've updated default bitcoind docker image to 0.17 as well as for the multiwallet.

A lot of tests broke with this update, I've fixed many of them and removed some that didn't make sense anymore, this may need some work.

I still need to add 'signrawtransactionwithkey', 'sethdseed' these two commands, possibly a signrawtransaction change (I believe there's an issue for that).

Need to update documentation as well.

### CODE REVIEW

File: dist/src/methods.js [hunk 1]
```diff
@@ -41,18 +41,27 @@ var _default = {
   clearBanned: {
     version: '>=0.12.0'
   },
+  combinePsbt: {
```
Comment (pedrobranco) [inline dist/src/methods.js]: Discard all changes in `dist` files.

**Comment by pedrobranco** [comment]
Thanks for the pull request @juscamarena. 

Can we create an initial commit only with the `src/methods` which are required for the 0.17 version? Then we can add other commits on top of that for bumping the containers versions and fixing all the tests.

**Comment by juscamarena** [comment]
Sure thing.

**Comment by juscamarena** [comment]
I've combined the container update and test commits together, I can separate them out as needed, there are probably some more tests that can be added as well.

**Comment by juscamarena** [comment]
1 test fails, there are a few missing methods.

One is quite peculiar getaddressbyaccount, I've never seen this one before, rpc method is getaddressesbyaccount not getaddressbyaccount so not sure why tests fails there.

The other PR includes signrawtransactionwithkey so I didn't include it here.

**Comment by juscamarena** [comment]
Looks like no interest. I'll just start making changes to my own forked version of this.

**Comment by pedrobranco** [comment]
> Looks like no interest. I'll just start making changes to my own forked version of this.

I'm very sorry about the delay but we haven't much time lately. I will pick this right now.

**Comment by pedrobranco** [comment]
Replaced by #94 which is rebased and fixed a failing test. Thanks @juscamarena.

**Pull request closed without merging**